datasources/pom-template.xml

```diff
@@ -19,12 +19,21 @@
     data sources contacted thought their REST API.
   </description>
 
+  <properties>
+    <destDirectory>datasource</destDirectory>
```

Comment:
the plugin is with a `s`, we should keep it consistant everywhere.

Reply:
Yep